Tutorial_1c_Line_Ratios.ipynb

```diff
@@ -0,0 +1,736 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.    plt.step(xx,kde_ratio(xx))
</pre><p>I was unsure why the KDE was needed if you end up binning the data. Is there a reason that's preferable over using <code>xx</code> as the input bins for <code>plt.hist</code>? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Smithsonian/sma-science-tutorials/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>